tsdown.config.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  defineConfig,
+} from 'tsdown';
+
+export default defineConfig({
+  attw: {
+    level: 'error',
+    profile: 'node16',
+  },
+  dts: {
+    sourcemap: true,
+  },
+  entry: [
+    './src/index.js',
+    './src/getJsdocProcessorPlugin.js',
+    './src/iterateJsdoc.js',
+  ],
+  format: [
+    'es',
+    'cjs',
+  ],
+  outputOptions: {
+    exports: 'named',
+  },
+  publint: {
+    level: 'error',
+    strict: true,
+  },
+  sourcemap: true,
+  unbundle: true,
```

Comment:
I chose to keep the dist similar to the old dist.
If you'd prefer to bundle the dist, happy to change it.